documentation/src/main/asciidoc/topics/con_store_migrator.adoc

```diff
@@ -14,3 +14,9 @@ the target cache.
 `StoreMigrator` also lets you migrate data from one type of cache store to
 another. For example, you can migrate from a JDBC String-Based cache store to a
 Single File cache store.
+
+[IMPORTANT]
+====
+`StoreMigrator` migrates non-segmented cache stores to segmented cache stores
+only. It is not currently possible to migrate segmented cache stores.
+====
```

Comment:
@ryanemerson hey does this sound right? not sure if we need to add some guidance on what to do if you have a segmented cache store with an older version, such as 9.4. I think it's probably not an issue for RHDG given that segmentation was not supported with most cache stores, if any. but could there be a case where users in 9.4 could have a segmented cache store and no path to migrate data to 10+?

Reply:
@wburns When were segmented stores added? 9.4.x or 10.x?

Reply:
It was added in 9.4.x. The sentence here is incorrect. The problem is you can't migrate from a segmented store to either different number of segment store or non segmented.

Reply:
thanks for clarifying @wburns!